judge/bridge/judge_list.py

```diff
@@ -52,6 +52,9 @@ def _handle_free_judge(self, judge):
                         break
                 node = node.next
 
+    def count_not_disabled(self):
+        return len(self.judges) - sum(judge.is_disabled for judge in self.judges) > 1
```

Comment:
You're treating this like an `int` on line 36, but returning a `bool` here

Reply:
Oh true, the `>1` must've slipped when writing the function.